coala_quickstart/generation/Utilities.py

```diff
@@ -187,3 +188,51 @@ def get_language_from_hashbang(hashbang):
             # For eg: #!bin/bash
             hashbang_element = hashbang_contents[0].split('/')
             return (hashbang_element[len(hashbang_element)-1])
+
+
+def concatenate(dict1, dict2):
+    """
+    Concatenates 2 dicts of the type:
+    eg.
+
+    dict1:
```

Comment:
this needs to be a doctest.

c.f. https://github.com/coala/coala-quickstart/pull/263#discussion_r208909218